ui/app/src/Views.elm

```diff
@@ -16,12 +16,31 @@ import Views.NavBar.Views exposing (navBar)
 view : Model -> Html Msg
 view model =
     div []
-        [ navBar model.route
+        [ renderLink
+        , fontAwesome
+        , navBar model.route
         , div [ class "container pb-4" ]
             [ currentView model ]
         ]
 
 
+renderLink : Html msg
+renderLink =
+    node "link"
+        [ href "https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css"
```

Comment:
+1 to @mxinden's suggestion, we should be doing that regardless. But perhaps in a different pr :)